web/LICENSE

```diff
@@ -1,5 +1,6 @@
 Copyright 2017 Drone.IO Inc
 Copyright 2019 Laszlo Fogas
+Copyright 2020 Woodpecker Authors
```

Comment:
hmm anbraten did rewrote the whole web frontend

@anbraten did you use some old parts ?

else we can remove the other two rows